protocol/x/clob/abci.go

```diff
@@ -117,6 +128,11 @@ func PrepareCheckState(
 	ctx sdk.Context,
 	keeper *keeper.Keeper,
 ) {
+	ctx = log.AddPersistentTagsToLogger(ctx,
+		log.Handler, log.PrepareCheckState,
+		log.BlockHeight, ctx.BlockHeight(),
```

Comment:
Shouldn't this be block height + 1 since it's `PrepareCheckState` for the next block?